vllm_batch_inference/job.sub

```diff
@@ -0,0 +1,33 @@
+JobBatchName            = "vLLM batch inference template"
+
+universe                = docker
+docker_image            = ghcr.io/jasonlo/vllm-bash:v0.8.1
+
+# Artefact
+executable              = run.sh
+transfer_input_files    = .env, batch_inference.py, inputs.jsonl
+transfer_output_files   = outputs.jsonl
+should_transfer_files   = YES
+
+# Logging
+stream_output           = true
+output                  = condor_log/output.$(Cluster)-$(Process).txt
+error                   = condor_log/error.$(Cluster)-$(Process).txt
+log                     = condor_log/log.$(Cluster)-$(Process).txt
+
+# Compute resources
+request_cpus            = 2
+request_memory          = 8GB
+request_disk            = 10GB
+
+# Extra GPU settings
+request_gpus            = 1
+
+# (Optional) Depending on the model and batch size, request GPU with sufficient memory   
+require_gpus            = GlobalMemoryMb >= 20000
+Requirements            = (Target.CUDADriverVersion >= 10.1)
```

Comment:
This won't filter anything within CHTC, so I suggest we remove.

```
13:01:04 @deepdivesubmit2000 ~/templates-GPUs/vllm_batch_inference (pr31✗) » condor_status -gpus -af CUDADriverVersion | sort | uniq -c
      2 11.6
    300 12.4
      6 12.6
      3 12.8
  12735 undefined
```

Reply:
Might it be a good habit to know and set these requirements even if they don't currently have an effect in the target pool? I see a case for leaving it.

Reply:
That's a fair point. It's not one of the handles we regularly educate users on (and adds yet another dimension to the CUDA toolkit/CUDA driver/device driver/compatibility confusion), but I'm ok with keeping it.